components/data-bridge/org.wso2.carbon.databridge.agent/src/main/java/org/wso2/carbon/databridge/agent/DataPublisher.java

```diff
@@ -143,7 +143,8 @@ private void processEndpoints(DataEndpointAgent dataEndpointAgent,
             boolean failOver = (Boolean) receiverGroup[0];
 
             DataEndpointGroup endpointGroup;
-            if (failOver) endpointGroup = new DataEndpointGroup(DataEndpointGroup.HAType.FAILOVER, dataEndpointAgent);
+            if (failOver)
```

Comment:
add {} after if (failOver), based on WSO2 cording standard it has to be if (failOver) { ... }